packages/side-nav/src/SideNavItem.tsx

```diff
@@ -212,7 +181,10 @@ const RootComponentTypes = {
  *  by this prop. Other additional props will be spread on the anchor element.
  */
 
-const SideNavItem = React.forwardRef((props: SideNavItemProps, forwardRef) => {
+const SideNavItem: ExtendableBox<
```

Comment:
Aware that this change might warrant a larger version bump, lmk your thoughts @lazytype

Reply:
wouldn't hurt!

Reply:
👍